infra/modules/providers/azure/service-principal/variables.tf

```diff
@@ -1,7 +1,7 @@
 variable "create_for_rbac" {
   description = "Create a new Service Principle"
   type        = bool
-  default     = true
+  default     = false
 }
 
 variable "display_name" {
```

Comment:
Can we give this a default name? ex. ("RBAC Service Principle") Right now, clients using the module are forced to provide a "display name" even if it won't be used.

Reply:
ok I'd file a bug on this.